account_operating_unit/README.rst

```diff
@@ -0,0 +1,109 @@
+.. image:: https://img.shields.io/badge/license-LGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/lgpl.html
+   :alt: License: LGPL-3
+
+===============================
+Accounting with Operating Units
+===============================
+
+This module introduces the following features:
+
+* Adds the Operating Unit (OU) to the account move line.
+
+* Defines if the Operating Units are self-balanced and Inter-Operating Unit
+  clearing account at company level.
+
+* Journal entry with lines in different Operating Units are checked based on
+  the "self-balanced" set up in OU.
+
+  At the time of posting the journal entry, the corresponding lines in the
+  Inter-Operating Unit clearing account are automatically created, making
+  each OU self-balanced.
+
+* The account financial reports include the option to filter by OU.
+
+* Adds the Operating Unit (OU) to the invoice.
+
+* Implements security rules in the invoice based on OU.
+
+Installation
+============
+
+No specific installation requirements.
+
+Configuration
+=============
+
+If your company is required to generate a balanced balance sheet by
+Operating Unit you can specify at company level that Operating Units should
+be self-balanced, and then indicate a self-balancing clearing account.
+
+* Create an account for "Inter-OU Clearing" of type Regular.
+
+* Go to *Settings / Companies / Companies* and Set the "Operating Units are
+  self-balanced" checkbox.
+
+  Then Set the "Inter-OU Clearing"  account in "Inter-Operating Unit
+  clearing account" field.
+
+* Assign Operating Unit in Accounts.
+
+
+Usage
+=====
+
+* Add the Operating Unit to invoices.
```

Comment:
This section is not clear enough: it looks like a list of features mixed with procedure.